cmd/tailscale/cli/up.go

```diff
@@ -663,6 +668,10 @@ func runUp(ctx context.Context, cmd string, args []string, upArgs upArgsT) (retE
 		if err != nil {
 			return err
 		}
+		authKey, err = resolveAuthKey(ctx, authKey)
+		if err != nil {
+			return err
+		}
```

Comment:
this should also work for `tailscale login` not just for `tailscale up`

Reply:
oh does it already?

Reply:
the GitHub action only uses `up` and it's experimental for now, aimed at the GitHub Action primarily.

But sure.